conf/cassandra.yaml

```diff
@@ -1358,3 +1358,64 @@ enable_sasi_indexes: false
 # Enables creation of transiently replicated keyspaces on this node.
 # Transient replication is experimental and is not recommended for production use.
 enable_transient_replication: false
+
+# Apply database-as-a-service defaults.
```

Comment:
Since this flag won't make it into OSS we need to remove it and ask Jim to apply these directly.

Reply:
as discussed on Slack: We'll port guardrails "mostly" as they are in DSE and then later apply any additional refactoring that is required for OSS